src/core_plugins/kibana/ui_setting_defaults.js

```diff
@@ -226,6 +226,12 @@ export function getUiSettingDefaults() {
       value: '($0,0.[00])',
       description: 'Default <a href="http://numeraljs.com/" target="_blank">numeral format</a> for the "currency" format'
     },
+    'format:number:defaultLocale': {
+      value: 'en',
+      type: 'select',
+      options: ['be-nl', 'chs', 'cs', 'da-dk', 'de-ch', 'de', 'en', 'en-gb', 'es-ES', 'es', 'et', 'fi', 'fr-CA', 'fr-ch', 'fr', 'hu', 'it', 'ja', 'nl-nl', 'pl', 'pt-br', 'pt-pt', 'ru-UA', 'ru', 'sk', 'th', 'tr', 'uk-UA'],
```

Comment:
We should be able to replace this array with the ids from `@elastic/numeral/languages` right?

Reply:
Yes, it would be great to dynamically inject all language ids from "languages.js" array. 
That said, this configuration file seems to be completely static. 
=> do you know how to dynamically object values here ?

Reply:
You can require files at the top just like everything else, it's not actually static, just doesn't have any dependencies ***yet***.

Reply:
You're right, sorry for this remark. Even more, I just saw that there is already one dependency. 
I will fix this file to dynamically inject array from "languages.js".